import-api/mastr_storage.json

```diff
@@ -0,0 +1,162 @@
+            {"name": "PraequalifiziertFuerRegelenergie", "description": "Einheit ist für die Vorhaltung und Erbringung von Regelenergie präqualifiziert", "type": "string", "unit": "none"},
+            {"name": "Einzatzort", "description": "An welchem Einsatzort wird die Stromerzeugungseinheit als Notstromaggregat betrieben", "type": "string", "unit": "none"},
+	    {"name": "AcDcKoppelung", "description": "AC oder DC gekoppeltes System", "type": "string",	"unit": "null"},
+	    {"name": "Batterietechnologie", "description": "Batterietechnologie", "type": "string", "unit": "null"},	
+	    {"name": "PumpbetriebLeistungsaufnahme", "description": "Höchste elektrische Dauerleistung, die dem  Netz für den Pumpbetrieb unter Nennbedingungen entnommen werden kann", "type": "string", "unit": "null"},
+	    {"name": "PumpbetriebKontinuierlichRegelbar", "description": "Die Wasserkraftanlage ist im Pumpbetrieb kontinuierlich regelbar", "type": "string", "unit": "null"},
+	    {"name": "Pumpspeichertechnologie", "description": "Technologie des Pumpspeichers", "type": "string", "unit": "null"},
+	    {"name": "Notstromaggregat", "description": "Angabe ob die Stromerzeugungseinheit zur Versorgung bei Stromnetzstörungen dient (Notstromaggregat)", "type": "string", "unit": "null"},
+	    {"name": "BestandteilGrenzkraftwerk", "description": "Gehört die Stromerzeugungseinheit zu einem Grenzkraftwerk", "type": "string",	"unit": "null"},
+	    {"name": "NettonennleistungDeutschland", "description": "Die höchste elektrische Dauerleistung unter Nennbedingungen dieser Stromerzeugungseinheit, die in ein deutsches Stromnetz eingespeist werden kann.", "type": "string", "unit": "null"},
+	    {"name": "ZugeordnenteWirkleistungWechselrichter", "description": "Zugeordnente Wirkleistung des/der Wechselrichter", "type": "string", "unit": "null"},
+	    {"name": "NutzbareSpeicherkapazitaet", "description": "Nutzbare Speicherkapazität", "type": "string", "unit": "null"}
+	    {"name": "SpeMastrNummer", "description": "MaStR-Nummer der verknüpften Speicheranlage", "type": "string",	"unit": "null"},
+	    {"name": "EegMastrNummer", "description": "MaStR-Nummer der zugeordneten EEG-Anlage", "type": "string", "unit": "null"},
+	    {"name": "EegAnlagentyp", "description": "Typ der Einheit",	"type": "string", "unit": "null"},
+	    {"name": "Technologie", "description": "Technologie der Stromspeicherung",	"type": "string", "unit": "null"}
+ 	    {"name": "version_e", "description": "Version (eeg)", "type": "string", "unit": "none"},
+            {"name": "timestamp_e", "description": "Timestamp (eeg)", "type": "timestamp", "unit": "none"} ],
+	 "primaryKey": "id"},
+    "dialect":
+        {"delimiter": ";",
+        "decimalSeparator": "."} } ],
+"review": {
+    "path": "https://github.com/OpenEnergyPlatform/data-preprocessing/issues/13",
+    "badge": "none"},
+"metaMetadata":
+    {"metadataVersion": "OEP-1.4",
```

Comment:
It seems like the MaStR metadata is still with the "old" version 1.4. There was a final release as [OEP-1.4.0](https://github.com/OpenEnergyPlatform/examples/blob/master/metadata/oep_metadata_template.json#L80)
We will merge this PR and start a new branch to update all metadata to the newest version.

Reply:
Created issue #43 for that :)